config/subsidy-application-management/forms/urban-renewal/application-phase2/versions/20221218090000/form.ttl

```diff
@@ -485,9 +490,464 @@ ext:ProjectfaseringMetProjectverloopTimingEnRisicobeheerProjectGroup
+        sh:group          ext:projectOnderdeelPg-sub .
+
+
+    ##########################################################
+    # Field: Projectonderdeel toelichting Input
+    ##########################################################
+
+    ext:projectOnderdeelBeschrijvingF
+        a                 form:Field ;
+        sh:name           "Geef een korte toelichting over wat dit projectonderdeel inhoudt" ;
+        sh:order          1 ;
+        sh:path           lblodSubsidie:projectOnderdeelBeschrijving ;
+        form:validations
+                          [ a                 form:MaxLength ;
+                            form:grouping     form:MatchEvery ;
+                            form:max          "750" ;
+                            sh:resultMessage  "Max. karakters overschreden." ;
+                            sh:path            lblodSubsidie:projectOnderdeelBeschrijving
+                          ],
+                          [ a                 form:RequiredConstraint ;
+                            form:grouping     form:Bag ;
+                            sh:resultMessage  "Dit veld is verplicht." ;
+                            sh:path           lblodSubsidie:projectOnderdeelBeschrijving            
+                          ] ;
+        form:displayType  displayTypes:textArea ;
+        sh:group          ext:projectOnderdeelPg-sub .
+
+
+    ##########################################################
+    # Listing: Projectonderdeel Timing
+    ##########################################################
+        ##########################################################
+        #  property-group
+        ##########################################################
+        ext:projectOnderdeelTimingListingPg a form:PropertyGroup;
+            sh:name "Timing" ;
+            form:help """Geef de verschillende stappen aan, zoveel mogelijk in chronologische volgorde, en duidt in de laatste twee kolommen de start- en einddatum aan.""";
+            sh:order 1 .
+
+        ##########################################################
+        #  listing-scope: timing
+        ##########################################################
+        ext:projectOnderdeelTimingListingS a form:Scope;
+          sh:path lblodSubsidie:projectOnderdeelTimingUnit.
+
+        ##########################################################
+        # Listing: Projectonderdeel timing listing
+        ##########################################################
+        ext:projectOnderdeelTimingL a form:Listing;
+          a form:ListingTable;
+          form:each ext:projectOnderdeelTimingListingSf;
+          form:scope ext:projectOnderdeelTimingListingS;
+          form:createGenerator ext:projectOnderdeelTimingListingGenerator;
+          form:canAdd true;
+          form:addLabel "Voeg stap toe";
+          form:canRemove true;
+          form:removeLabel "Verwijder";
+          form:tableIndexLabel "#";
+          form:showTableRowIndex true;
```

Comment:
We still need to bump the version in the frontend I think.

Reply:
did that locally, will probably push the bump straight to dev branch